content/en/docs/configuration/acme/dns01/akamai.md

```diff
@@ -5,27 +5,85 @@ weight: 30
 type: "docs"
 ---
 
-## Akamai FastDNS
+## Edge DNS
+
+Use Edge DNS to solve DNS01 ACME challenges by creating a `Secret` using [Akamai API credentials](https://developer.akamai.com/getting-started/edgegrid) and an `Issuer` that references the `Secret` and sets the  solver type.
+
+### Create a Secret
+
+The `Secret` should look like the following for the `Issuer` to reference. Replace `use_akamai_client_secret`, `use_akamai_access_token` and `use_akamai_client_token` with the respective Akamai API credential values.
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: akamai-secret
+type: Opaque
+stringData:
+  clientSecret: use_akamai_client_secret
+  accessToken: use_akamai_access_token
+  clientToken: use_akamai_client_token
+```
+
+### Create an Issuer
+
+To set Edge DNS for challenge tokens, `cert-manager` uses an `Issuer` that references the above `Secret` and other attributes such as the solver type. The `Issuer` should look like the following. Replace `use_akamai_host` with the Akamai API credential `host` value.
 
 ```yaml
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
-  name: example-issuer
+  name: letsencrypt-akamai-dns
 spec:
   acme:
-    ...
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: contact@me.com
+    privateKeySecretRef:
+      name: letsencrypt-akamai-issuer-account-key
     solvers:
     - dns01:
         akamai:
-          serviceConsumerDomain: akab-tho6xie2aiteip8p-poith5aej0ughaba.luna.akamaiapis.net
+          serviceConsumerDomain: use_akamai_host
           clientTokenSecretRef:
-            name: akamai-dns
+            name: akamai-secret
             key: clientToken
           clientSecretSecretRef:
-            name: akamai-dns
+            name: akamai-secret
             key: clientSecret
           accessTokenSecretRef:
-            name: akamai-dns
+            name: akamai-secret
             key: accessToken
 ```
+
+### Create a Certificate
+
+The `Certificate` should look like the following and reference the Akamai Edge DNS `Issuer` above.
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: example-zone
+spec:
+  secretName: akamai-crt-secret
+  dnsNames:
+  - '*.example.zone'
+  issuerRef:
+    name: letsencrypt-akamai-dns
+    kind: Issuer
+```
+
+> Note: `cert-manager` will wait for challenge tokens to propagate across the Edge DNS network. Follow the `certificate` status with a command such as the following.
+
+```bash
+kubectl describe certificate example.zone. -n cert-manager
+```
+
+### Troubleshooting
+
+Follow the `cert-manager` logs to identify any issues. These commands will show the running `cert-manager` pod and its logs assuming a namespace of `cert-manager`.
+
+```bash
+kubectl get pods -n cert-manager
+kubectl logs cert-manager-dddddddddd-ddddd -n cert-manager
+```
```

Comment:
We aim to make all relevant diagnostic messages available as events, linked to cert-manager custom resources. 
So users should not need to follow the logs of cert-manager pods.

I suggest you link to the `kubectl cert-manager status certificate` CLI which will print all the events and conditions of a certificate and its various dependency resources:  https://cert-manager.io/docs/usage/kubectl-plugin/#status-certificate

Reply:
Good suggestion. Thanks.